Makefile

```diff
@@ -1,13 +1,16 @@
 ZPOOL=""
 SERVER=""
-PYTHON?=/usr/local/bin/python3
+PYTHON?=/usr/local/bin/python3.6
 
-install:
-	@test -s ${PYTHON} || (echo "Python binary ${PYTHON} not found, please enter a valid one"; exit 1)
-	@(pkg -vv | grep -e "url.*/latest") > /dev/null 2>&1 || (echo "Please ensure pkg url is using \"latest\" instead of \"quarterly\" in /etc/pkg/FreeBSD.conf before proceeding with installation"; exit 1)
+depends:
+	@(pkg -vv | grep -e "url.*/latest") > /dev/null 2>&1 || (echo "It is advised pkg url is using \"latest\" instead of \"quarterly\" in /etc/pkg/FreeBSD.conf.";)
```

Comment:
This was originally added since python3.6 was only in latest, I believe this is in quarterly now, so we can probably just drop this test

Reply:
Actually there's another usecase - py-libzfs. Some tests would fail if py-libzfs is being used with the quarterly version. There still might be some bugs as well which have been addressed in latest ( an assumption for the bug one )

Reply:
Please advise on how to proceed. I think we should keep this warning in this case

Reply:
Agreed :)